packages/expo-router/src/modal/Modal.tsx

```diff
@@ -30,6 +31,14 @@ export interface ModalProps extends ViewProps {
    * Callback that is called after modal is shown.
    */
   onShow?: () => void;
+  /**
+   * Callback that is called after detent change.
+   * Works only when `presentation` is set to `formSheet`.
+   *
+   * @platform ios
+   * @platform android
+   */
```

Comment:
Can you add a note that this is not fired when modal is open? So that people know that this will not be fired when modal is reopened 

https://github.com/user-attachments/assets/57762a25-9fee-468e-b3e3-55235384e0f5

Reply:
hmm it's actually just a bug on the example. It should be called when every time the detent is changed by dragging.
Will fix example instead.